packages/components/carousel/src/carousel.vue

```diff
@@ -300,6 +305,13 @@ watch(
   }
 )
 
+watch(
+  () => props.interval,
+  () => {
+    resetTimer()
```

Comment:
Maybe you can call `pasueTimer` and `startTimer` directly without creating the `resetTimer` function

```suggestion
    pauseTimer()
    startTimer()
```

Reply:
Hi, I think the timer needs to be reset when calling the `setActiveItem` method too. What do you think about calling `pauseTimer` and `startTimer`?

Reply:
You are right 👍